OrbitVulkanLayer/QueueManager.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "QueueManager.h"
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit_vulkan_layer {
+
+void QueueManager::TrackQueue(const VkQueue& queue, const VkDevice& device) {
```

Comment:
VkQueue and VkDevice seem to be pointers. Does it make sense to pass them by value?

Reply:
This will occur in so many places in the Vulkan layer, let's think about it well now.

Reply:
Right they are typedefs of `VkQueue_T*` and `VkDevice_T*`. I thought of passing them as values, but then always find this confusing to see objects passed by value, where you not directly see that they are actually just pointers. I am not too strong about this though.

Reply:
As discussed offline, I went with passing them by value.